### DISCUSSION TITLE
Practica del modulo HTML - Mazun

### DISCUSSION THREAD

**Discussion by LaMazorca** [Launch-X-Latam/MisionFrontEnd#249] [Show and tell]
Compañeros les dejo mi practica del HTML para compartirles y me apoyen con sus retroalimentaciones, les agradezco

[Practica](https://lamazorca.github.io/)

**Comment by JohnnChavez** [comment]
Como estructura HTML lo ve bien, también hay una practica del modulo HTML sobre una pastelería; es [este por si no lo has visto](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas)

**Reply by LaMazorca** [reply]
gracias compañero eso estuve viendo que hablaban sobre eso pero no encontraba dónde estaba, te agradezco el aporte.
